api/app/fireweather_bot/noon_forecasts.py

```diff
@@ -23,6 +23,18 @@
 
 LOGGER = logging.getLogger(__name__)
 
+"""
+Define a "fire season" applicable to all weather stations.
+If bot fails to retrieve a forecast for any weather station during the fire season,
+an automatic notification will be sent via Rocketchat.
+Outside of the fire season, forecasts are not usually issued, so no RC notification
+should be sent if the bot fails to retrieve any data.
+"""
+FIRE_SEASON_START_MONTH = 4
```

Comment:
If this were a long term solution, I'd say we should but this in config - but since this all goes away the moment we have a working API....

Reply:
agreed. I considered moving it to our config map, but then I would have had to add in error handling for invalid inputs and blah blah blah. Not worth it